source/_components/sensor.vasttrafik.markdown

```diff
@@ -29,16 +29,42 @@ sensor:
       - from: Musikvägen
 ```
 
-Configuration variables:
-
-- **key** (*Required*): The API key to access your Västtrafik account.
-- **secret** (*Required*): The API secret to access your Västtrafik account.
-- **departures** array (*Required*): List of travel routes.
-  - **name** (*Optional*): Name of the route.
-  - **from** (*Required*): The start station.
-  - **heading** (*Optional*): Direction of the traveling.
-  - **lines** (*Optional*): Only consider these lines.
-  - **delay** (*Optional*): Delay in minutes. Defaults to 0.
+{% configuration %}
+key:
+  description: "The API key to access your Västtrafik account."
+  required: true
+  type: string
+secret:
+  description: "The API secret to access your Västtrafik account."
+  required: true
+  type: string
+departures:
+  description: "List of travel routes."
+  required: true
+  type: map
+  keys:
+    name:
+      description: "Name of the route."
+      required: false
+      type: string
+    from:
+      description: "The start station."
+      required: false
```

Comment:
`from` is required, according to the original version.

Reply:
Thank you. Should be resolved by by c363021